app/javascript/components/batch_update/components/App.js

```diff
@@ -13,7 +13,7 @@ import { Notices, Notice } from './Notices'
 const findByName = (items, item) => items.find(i => i.name === item.name)
 const findById = (items, item) => items.find(i => i.id === item.id)
 
-const commonGenesToIgnore = ['Art', 'Career Stage Gene']
+const commonGenesBlacklist = ['Art', 'Career Stage Gene']
```

Comment:
As configured, this would result in the following ESLint error:

<img width="1556" alt="Screen Shot 2021-12-01 at 1 04 44 PM" src="https://user-images.githubusercontent.com/140521/144687013-8c1c42b2-b0af-4921-82ab-2bb3a5b66e49.png">